code/modules/projectiles/guns/energy/special.dm

```diff
@@ -569,16 +569,19 @@
 	w_class = WEIGHT_CLASS_BULKY
 	fire_sound = 'sound/weapons/pulse3.ogg'
 	desc = "A gun that changes the body temperature of its targets."
-	var/temperature = 300
-	var/target_temperature = 300
 	origin_tech = "combat=4;materials=4;powerstorage=3;magnets=2"
 
 	ammo_type = list(/obj/item/ammo_casing/energy/temp)
 	selfcharge = TRUE
 
-	var/powercost = ""
-	var/powercostcolor = ""
-	var/dat = ""
+	// Measured in Kelvin
+	var/temperature = T20C
+	var/target_temperature = T20C
+	var/min_temp = 0
+	var/max_temp = 500
+
+	var/powercost = "Low"
```

Comment:
this method of displaying power cost should be moved out of DM and into the TGUI menu